### PR TITLE
feat: ui regression suite initial commit

### PR DESCRIPTION
**Pull request by nalmabrcom**
Regression UI test suite.
## How to test
1. Run `yarn install`
2. Run 
<code>.\node_modules\.bin\tsc --build .\node_modules\@eclipse\che-che4z\tests\tsconfig.json</code>
 inside `tests` folder. 
 3. Prepare your Theia environment (I'm using `theia-java:1.5.0` docker container). 
 4. Run
 <code> .\node_modules\.bin\cypress run --spec '.\cypress\integration\LSP\*' --browser chrome  --config baseUrl=https://theia --headless </code>
 
## Results

```

====================================================================================================

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ √  LSP\copybook.spec.js                     04:09       20       17        -        3        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F101836.spec.js                      00:11        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F102470.spec.js                      00:16        3        2        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F95833.spec.js                       00:35        5        4        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F95834.spec.js                       00:10        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F96117.spec.js                       00:38        9        6        -        3        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F96125.spec.js                       00:55        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\F97476.spec.js                       01:16       10        4        -        6        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √  LSP\lsp.spec.js                          04:47       23       20        -        3        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    √  All specs passed!                        13:00       77       60        -       17        -
```
    
## Additional information
In further PRs should be resolved skipped tests. 

Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

**Review by temanbrcom — COMMENTED** [review]
Looks good, but the copyright header is missing in some files

**Comment by temanbrcom** [comment]
Is it possible to run the tests using IDEA run configurations?

**Comment by nalmabrcom** [comment]
> Is it possible to run the tests using IDEA run configurations?

Honestly never tried in IDEA, but in theory, should be fine.

**Comment by nalmabrcom** [comment]
> Looks good, but the copyright header is missing in some files

Oh, I see those files. It will impact most of the tests, but I'll work on that.

File: tests/test_files/project/CALC-DATA.cbl [hunk 1]
```diff
@@ -0,0 +1 @@
+s
```
Comment (temanbrcom) [inline tests/test_files/project/CALC-DATA.cbl:1]: why is this one empty?

**Review by temanbrcom — APPROVED** [review]
☁️

**Review by grianbrcom — APPROVED** [review]
Tested it locally. It works overall, but ~8..10 tests are failing.

File: tests/test_files/project/CALC-DATA.cbl (hunk 1 above)
Comment (nalmabrcom) [reply tests/test_files/project/CALC-DATA.cbl]: This file was created automatically in tests. Should better cleanup those files.

**Pull request merged**